drivers/hmis_csv_twenty_twenty_four/app/models/hmis_csv_twenty_twenty_four/exporter/export_concern.rb

```diff
@@ -110,12 +110,28 @@ def self.csv_header_override(keys)
     end
 
     def process(row)
+      row = enforce_lengths(row)
       row = assign_export_id(row)
       row = self.class.adjust_keys(row)
 
       row
     end
 
+    def enforce_lengths(row)
+      length_limited_columns.each do |k, opts|
+        next if row[k].blank? || row[k].length <= opts[:limit]
+
+        row[k] = row[k][0..opts[:limit]]
```

Comment:
`(0..32).length == 33`, so I think this should include one too many chars?
```suggestion
        row[k] = row[k][0...opts[:limit]]
```
The 0 is optional, but maybe clearer than `...opts[:limit]`?

Reply:
Thanks!